modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -8,34 +8,19 @@ These limitations include:
 * No downloading
 * No editing
 * Watermarking
-+
-NOTE: Administrators can configure the watermark text.
-* Optional printing and exporting to PDF with watermarks included
+* Optional printing and exporting to PDF with watermarks included, which can be adjusted
```

Comment:
Saying "Optional printing and exporting to PDF" seems redundant. What about just "Optional exporting to PDF..."?

Reply:
It was you who wrote this, now you say it's redundant?

This is the admin manual - so for admins. I expect that is known that everything here is for admins.